### DISCUSSION TITLE
Business does not take goods from Storage

### DISCUSSION THREAD

**Discussion by AWoelfel** [voxeltycoon/issues#675] [General] [answered]
**Describe the bug**
I have a business "Hardware Store" (taking "Wood Planks") which is in reach of Freight Station "Cincinnati South". The Station is in reach of a Warehous in which "Wood Planks" are stored. The Station shows the "Wood Planks" as available for distribution "AVAILABLE IN BUILDINGS" but does not foreward the goods to the "Hardware Store".

![image](https://user-images.githubusercontent.com/8323739/116776472-e29a0f00-aa68-11eb-9f40-f209b876e9f4.png)

Found a workaround : 
I than bought a Transport Car and set it up to take "Wood Planks" from "Cincinnati South" and deliver it to "Cincinnati South" which lead to the car being loaded and unloaded while parking in front of "Cincinnati South". During each unload the "Hardware Store" got its goods. This seems to be a working workaround.

![image](https://user-images.githubusercontent.com/8323739/116776483-eded3a80-aa68-11eb-9fb5-066930429055.png)

**Steps to reproduce**
- Build a Warehouse (W) and Freight Station (ST) in proximity to an consumer (C).
- Configure logistics to be   W <=> ST => C
- Fill Warehouse with needed (of C) Goods
- C's needs will not be fulfilled

**Build version**
0.85.1.0 Steam

**Platform**
Windows 10

**Reproducibility**
two out of two times so far 

**Logs/saves**
Attach [logs or saves](https://github.com/voxeltycoon/issues/discussions/650) if applicable.

**Comment by greeny** [comment]
The "available in buildings" display is to show you what is available for loading onto trucks/trains.

Stations aren't supposed to magically transfer cargo from warehouse to business. In tutorial, they teach you that you need to deliver cargo to a business using a truck.

Yes, you can use the "workaround" to do this.

**Comment by katepolak** [comment, marked as answer]
As @greeny says this is the intended behaviour. If you want this changed and have ideas for how it could be done please suggest them in the discussion about this over here: #475
